src/data-workspace/category-combo-table-body-pivoted/category-combo-table-body-pivoted.js

```diff
@@ -0,0 +1,174 @@
+        dataElements,
+        greyedFields,
+        /* 
+        filterText,
+        globalFilterText,
+        maxColumnsInSection,
+        renderRowTotals,
+        renderColumnTotals,*/
+        displayOptions,
+    }) {
+        const { data: metadata } = useMetadata()
+
+        const categories = selectors.getCategoriesByCategoryComboId(
+            metadata,
+            categoryCombo.id
+        )
+
+        const sortedCOCs = selectors.getSortedCoCsByCatComboId(
+            metadata,
+            categoryCombo.id
+        )
+
+        const categoryOptionsDetails = categories
+            .map((c) => {
+                const headerOptions = selectors.getCategoryOptionsByCategoryId(
+                    metadata,
+                    c.id
+                )
+                return [...headerOptions]
+            })
+            .flat()
+
+        const options = {
+            metadata,
+            categoryOptionsDetails,
+            sortedCOCs,
+            categories,
+            dataElements,
+        }
+
+        const rowsMatrix = generateFormMatrix(options, displayOptions)
+
+        return (
+            <>
+                {rowsMatrix.map((row, id /** todo: find suitable id */) => {
+                    return (
+                        <TableRow key={id}>
+                            {row.map((fieldInRow) => {
+                                if (
+                                    fieldInRow.type === 'columnHeader' ||
+                                    fieldInRow.type === 'rowHeader'
+                                ) {
+                                    return (
+                                        <TableCellHead
+                                            key={fieldInRow.id}
+                                            className={classNames(
+                                                [
+                                                    styles.pivotedHeader,
+                                                    styles.noWrap,
+                                                    styles[
+                                                        fieldInRow.metadataType
+                                                    ],
+                                                ],
+                                                {
+                                                    [styles.dataElementRowHeader]:
+                                                        fieldInRow.metadataType ===
+                                                        'dataElement',
+                                                }
+                                            )}
+                                            colSpan={fieldInRow.colSpan}
+                                            rowSpan={fieldInRow.rowSpan}
+                                        >
+                                            {fieldInRow.displayFormName !==
+                                                'default' &&
```

Comment:
I'm not sure if you want to use displayFormName here? It seems more logical to use formName. If there's a translation that is not default (e.g. into another language), the name will appear:
<img width="323" alt="image" src="https://github.com/dhis2/aggregate-data-entry-app/assets/18490902/33dcd109-5bc5-4ffd-8cdf-67f088e9d7a3">

Reply:
fixed